openpgp/write.go

```diff
@@ -435,27 +443,30 @@ type signatureWriter struct {
 	signer        *packet.PrivateKey
 	sigType       packet.SignatureType
 	config        *packet.Config
+	metadata      *packet.LiteralData // Protected by v5 keys only
```

Comment:
Protected -> Produced?

Reply:
I still don't understand the comment. :/
Is this available only in v5 keys?
Should this be protected only in v5 keys?

Reply:
Then the comment is definitely misleading, I will change it.
This metadata (format, filename, time) is hashed by v5 signatures, therefore protecting the integrity. In contrast, in a v4 signature, I am free to change the filename or creation time within the literal packet before the signature, and it will still validate against the sig.